apps/studio/components/interfaces/Docs/Snippets.ts

```diff
@@ -365,9 +387,14 @@ let { data: ${resourceId}, error } = await supabase
   .is('column', null)
   .in('column', ['Array', 'Values'])
   .neq('column', 'Not equal to')
+
   // Arrays
   .contains('array_column', ['array', 'contains'])
   .containedBy('array_column', ['contained', 'by'])
+
+  // Logical operators
+  .not('column', 'like', 'Negate filter')
+  .or('some_column.eq.Some value, other_column.eq.Other value')
```

Comment:
This right now needs raw PostgREST syntax as mentioned on https://supabase.com/docs/reference/javascript/or, which will look mysterious to users (all other methods are somewhat intuitive, except this one).

Maybe link to the above doc and additionally link to https://docs.postgrest.org/en/latest/references/api/tables_views.html#logical-operators on https://supabase.com/docs/reference/javascript/or.

Reply:
> Maybe link to the above doc

I think it would look a bit bloated with links in the examples, besides there's already a "Documentation" button (API docs in the Table editor) and a "Learn more" (API Docs from the sidebar) that link to the client docs. I think the examples should be a "quick reference", so if more info is needed then they can access those links.

> additionally link to https://docs.postgrest.org/en/latest/references/api/tables_views.html#logical-operators on https://supabase.com/docs/reference/javascript/or.

Good idea, can't find a link to the docs for PostgREST operators anywhere (the ["Match the filter"](https://supabase.com/docs/reference/javascript/filter) section should definitely have it).